firmware/user/display/sprite.h

```diff
@@ -5,9 +5,10 @@
 
 typedef struct
 {
-    const uint8_t  width;
-    const uint8_t  height;
-    const uint16_t data[16];
+    uint8_t  width;
+    uint8_t  height;
+    uint16_t data[16];
+    uint16_t dummy; // round size to multiple of 32 bits
```

Comment:
An alternative is to make a union with this struct and a `uint32_t[9]`. Or it could be decorated with `__attribute__ ((aligned (4)))`. So many ways to solve the same problem

Reply:
Richard. He is not fussed about how he did it